packages/commonwealth/server/util/requirementsModule/validateTopicGroupsMembership.ts

```diff
@@ -82,19 +71,6 @@ export async function validateTopicGroupsMembership(
     return { isValid: true };
   }
 
-  // check via old TBC without groups
-  try {
-    const canReact = await validateTopicThreshold(
-      tokenBalanceCacheV1,
-      models,
-      topicId,
-      address.address,
-    );
-    if (!canReact) {
-      return { isValid: false, message: Errors.InsufficientTokenBalance };
-    }
-    return { isValid: true };
-  } catch (e) {
-    throw new ServerError(Errors.BalanceCheckFailed, e);
-  }
+  // backwards compatibility with integration tests
+  return { isValid: true };
```

Comment:
Just for clarity's sake, can we invert the check here and return isValid = true immediately at the top if the flag is disabled, and then have the rest of the function beyond the flag check rather than inside it?